src/rgw/setup/rgw.py

```diff
@@ -101,9 +101,11 @@ def config(conf: MappedConf):
         if not os.path.exists(config_file):
             raise SetupError(errno.EINVAL, f'"{config_file}" config file is not present.')
 
+        Log.info('Generating SSL certs')
```

Comment:
will remove these logger statements during final deployment.
As of now added for debugging purpose only.

Reply:
done